Models/Department.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace BangazonAPI.Models
+{
+    public class Department
+    {
+        [Key]
+        public int DepartmentId { get; set; }
+
+        [Required]
+        public string DepartmentName { get; set; }
+        public int ExpenseBudget { get; set; }
+    }
```

Comment:
In ticket #7, it says: `If the query string parameter of ?_include=employees is provided, then all employees in the department(s) should be included in the response.`

Eventually we'll need a collection of employees to come back with each department. Let's go ahead and add that collection to the model so we have it when we need it.